Docs/source/index.rst

```diff
@@ -37,7 +37,6 @@ Keep the docs clear and readible. Reach out to more experienced team members wit
    Apps/Main
    Apps/ReadCarCAN
    Apps/ReadTritium
-   Apps/SendCarCAN
```

Comment:
Why is this removed?

Reply:
Whoops, I think that was a mistake. Thanks for catching that!